File: src/tasks/cli/feature/reword/RewordArguments.java
package tasks.cli.feature.reword;

import tasks.model.ObservableTask;

public final class RewordArguments {
  private final ObservableTask targetTask;
  private final String description;

  RewordArguments(ObservableTask targetTask, String description) {
    this.targetTask = targetTask;
    this.description = description;
  }

  /** The target to reword. */
  public ObservableTask targetTask() {
    return targetTask;
  }

  /** The new description for the task. */
  public String description() {
    return description;
  }
}

File: src/tasks/cli/feature/reword/RewordArguments.kt
package tasks.cli.feature.reword

import tasks.model.Task

class RewordArguments internal constructor(private val targetTask: Task, private val description: String) {
  /** The target to reword.  */
  fun targetTask(): Task {
    return targetTask
  }

  /** The new description for the task.  */
  fun description(): String {
    return description
  }
}